source/dotnet/Library/AdaptiveCards/AdaptiveRefresh.cs

```diff
@@ -23,8 +23,9 @@ public class AdaptiveRefresh
         [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
 #if !NETSTANDARD1_3
         [XmlElement(typeof(AdaptiveExecuteAction))]
+        [XmlElement(typeof(AdaptiveSubmitAction))]
 #endif
-        public AdaptiveExecuteAction Action { get; set; }
+        public AdaptiveAction Action { get; set; }
```

Comment:
Has this been changed? https://microsoft.visualstudio.com/AdaptiveCards/_build/results?buildId=96490456&view=results

Reply:
Teams will reject adaptive cards with Action.Execute when using TaskModule. It must be Action.Submit.  This means that this can be either ActionExecute or ActionSubmit, and Action is the base class.

Reply:
Hmmm....I don't have access to the link. My belief was that there is a teams bug where they reject cards with refresh of Action.Execute if you are doing a task module, but the documentation clearly says Action.Execute, so I will revert this.